### DISCUSSION TITLE
Need an indent before output

### DISCUSSION THREAD

**Discussion by cryingincradles** [spectreconsole/spectre.console#1226] [Q&A]
So, I'm trying to place Tree and Text. I need to make it so that there is a Text on the right and a Tree on the left. There seemed to be a solution, but I haven't found one. Tried creating a Grid, the output in the console is a ":", if you just output the columns, they are under each other, not in Text: Tree format

**Comment by patriksvensson** [comment]
@cryingincradles I'm not sure I understand what you're trying to do. Could you post some code or picture of what you're trying to achieve?
